automation/ContainerdTools.psm1

```diff
@@ -23,7 +23,7 @@ function Install-Containerd {
 
     $Version = $Version.TrimStart('v')
     Write-Output "* Downloading and installing Containerd v$version at $InstallPath"
-    "Downloading and installing Containerd v$version at $InstallPath" >> logs
+    "{0} - Downloading and installing Containerd v$version at $InstallPath" -f (Get-Date) >> logs
```

Comment:
The timestamp is a good addition. However, we do not need to redirect each message to a logger, since we can do this: `Install-Containerd >> logs.txt`. This way, the code will be much cleaner and we'll not need to remember to add `>> logs` to every log.